doc/api/util.md

```diff
@@ -68,8 +68,7 @@ exports.puts = util.deprecate(() => {
 It returns a modified function which warns once by default.
 
 If `--no-deprecation` is set then this function is a NO-OP.  Configurable
-at run-time through the `process.noDeprecation` boolean (only effective
-when set before a module is loaded.)
+at run-time through the `process.noDeprecation` boolean.
```

Comment:
@jasnell Just so we don’t talk past each other… 

```
This function does nothing if either the `--no-deprecation` command
line flag is used, or the `process.noDeprecation` property is set to
`true` *prior* to the first deprecation warning.
```

right? or am I misunderstanding either you or the way in which I’m proposing to change the behaviour here?